RoaringBitmap/src/test/java/org/roaringbitmap/buffer/TestMappeableArrayContainer.java

```diff
@@ -68,6 +68,33 @@ public void remove() {
     assertTrue(ac.contains((char) 1));
   }
 
+  @Test
+  public void removePresentValueFromReadOnlyContainer() {
```

Comment:
Please write similar tests in TestArrayContainer.java.

https://github.com/RoaringBitmap/RoaringBitmap/blob/master/RoaringBitmap/src/test/java/org/roaringbitmap/TestArrayContainer.java

We try to have the same test coverage. If you found a bug in MappeableArrayContainer.java, there might be one in ArrayContainer.java.